mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -67,7 +67,7 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 						"action": {
 							Type:         schema.TypeString,
 							Required:     true,
-							ValidateFunc: validation.StringInSlice([]string{"allow", "deny(403)", "deny(404)", "deny(502)"}, false),
+							ValidateFunc: validation.StringInSlice([]string{"allow", "deny(403)", "deny(404)", "deny(502)", "rate_based_ban", "threshold"}, false),
```

Comment:
Is "threshold" correct here? The documentation below and your example seem to indicate "throttle" is the expected value.

Reply:
Ahh sorry, it was a typo